dev-embedded/esptool/esptool-3.2-r1.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{8..10} )
+DISTUTILS_SINGLE_IMPL=1
+
+inherit distutils-r1
+
+DESCRIPTION="Utility to communicate with the ROM bootloader in Espressif ESP8266 and ESP32"
+HOMEPAGE="https://github.com/espressif/esptool"
+SRC_URI="https://github.com/espressif/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	$(python_gen_cond_dep '
+		dev-python/bitstring[${PYTHON_USEDEP}]
+		dev-python/cryptography[${PYTHON_USEDEP}]
+		>=dev-python/ecdsa-0.16.0[${PYTHON_USEDEP}]
+		dev-python/pyserial[${PYTHON_USEDEP}]
+		>=dev-python/reedsolomon-1.5.4-r1[${PYTHON_USEDEP}]
```

Comment:
You don't need to specify the version, USE flags will take care of it.

Reply:
Okay, I see.
That will make the -r1 release obsoloete - if its okay to change PYTHON_COMPAT in a stable package?

Reply:
@mgorny 
Important question - is it allowed to change 

```
-PYTHON_COMPAT=( python3_{8..9} )
+PYTHON_COMPAT=( python3_{8..10} )
```

in a stable ebuild?
This is the only effective change between `esptool-3.2.ebuild` and `esptool-3.2-r1.ebuild`.

Reply:
Yeah, that's fine.

Reply:
@thesamesam 
I though about having both esprool and reedsolomon change to -r1 - I think it's okay this way.
reedsolomon must go -r1 - there are more changes in it. So esptool must either follow -r1 or wait until reedsolomon ist stable again.
From my POV this PR is ready to commit.